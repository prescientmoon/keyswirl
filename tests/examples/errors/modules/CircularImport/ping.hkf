module exporting *

import errors.modules.CircularImport.pong
