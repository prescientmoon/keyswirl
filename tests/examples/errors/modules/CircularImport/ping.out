[error CircularImport]: Module "errors.modules.CircularImport.ping" implicitly imports itself
     +--> errors.modules.CircularImport.pong@3:8-3:42
     |
   3 | import errors.modules.CircularImport.ping
     :        ^---------------------------------
     :        `- This is where the cycle closes is
-----+

[error ModuleNotFound]: Module "errors.modules.CircularImport.ping" not found
     +--> errors.modules.CircularImport.pong@3:8-3:42
     |
   3 | import errors.modules.CircularImport.ping
     :        ^---------------------------------
     :        `- This module does not exist
-----+
